README.md

```diff
@@ -29,11 +29,11 @@ Hive users have a choice of 3 runtimes when executing SQL queries.
 Users can choose between Apache Hadoop MapReduce, Apache Tez or
 Apache Spark frameworks as their execution backend. MapReduce is a
 mature framework that is proven at large scales. However, MapReduce
-is a purely batch framework, and queries using it may experience
+is a pure batch framework, and queries using it may experience
 higher latencies (tens of seconds), even over small datasets. Apache
 Tez is designed for interactive query, and has substantially reduced
 overheads versus MapReduce. Apache Spark is a cluster computing
-framework that's built outside of MapReduce, but on top of HDFS,
+framework that's built outside MapReduce, but on top of HDFS,
```

Comment:
outside of is correct only, infact I feel both are correct.
https://english.stackexchange.com/questions/9700/outside-or-outside-of

Reply:
I will cancel this change